vmm/src/seccomp_filters.rs

```diff
@@ -120,6 +125,8 @@ fn create_vmm_ioctl_seccomp_rule() -> Result<Vec<SeccompRule>, Error> {
     Ok(or![
         and![Cond::new(1, ArgLen::DWORD, Eq, FIOCLEX)?],
         and![Cond::new(1, ArgLen::DWORD, Eq, FIONBIO)?],
+        and![Cond::new(1, ArgLen::DWORD, Eq, KVM_ARM_PREFERRED_TARGET,)?],
+        and![Cond::new(1, ArgLen::DWORD, Eq, KVM_ARM_VCPU_INIT,)?],
```

Comment:
These should have been gated by a condition based on the architecture.

Reply:
NP. I will create another commit to do it.

Reply:
Hmmm, checked other KVM APIs definitions, some other APIs are architecture specific as well: like KVM_GET_MSRS, KVM_GET_LAPIC. If these rules can distinguish with each other by it self (the const like `KVM_ARM_PREFERRED_TARGET` is  calculated by combining API ID, data structure size, etc.) and they don't affect each other, maybe we don't bother to gate them by arch-conditions. What do you think? @sboeuf

Reply:
I think we don't want potential guest to leverage the fact that some syscalls are available for other arch even when running on different architecture, as they might be able to trigger bugs and attack the host.
I agree that x86 specific syscalls should  also be gated properly.

Reply:
Agree, I will submit a PR to improve.
Thanks.

Reply:
Thanks @michael2012z